iphone/Maps/UI/BottomMenu/TabBar/BottomTabBarButton.swift

```diff
@@ -0,0 +1,33 @@
+import UIKit
+
+final class BottomTabBarButton: MWMButton {
+  @objc override func applyTheme() {
+    styleName = "BottomTabBarButton"
+    
+    for style in StyleManager.shared.getStyle(styleName)
+      where !style.isEmpty && !style.hasExclusion(view: self) {
```

Comment:
I was confused by the formatting. If the second line continues the first one, then the indent should be 4, not 2.
```suggestion
        where !style.isEmpty && !style.hasExclusion(view: self) {
```

Reply:
Fixed.